docs/examples/indie-linter-v2.md

```diff
@@ -0,0 +1,77 @@
+# Indie Linter v2
+
+The Indie Linter demoed below supports [Messages v2](../linter-message-v2.md) only.
+
+## package.json
+
+Only the `consumedServices` field is important in this manifest, the rest are just for reference
+
+```json
+{
+  "name": "my-indie-linter",
+  "main": "index.js",
+  "version": "0.0.0",
+  "private": true,
+  "description": "My Linter Package",
+  "keywords": ["linter-example", "example"],
+  "engines": {
+    "atom": ">=1.4.0 <2.0.0"
+  },
+  "consumedServices": {
+    "linter-indie": {
+      "versions": {
+        "2.0.0": "consumeIndie"
+      }
+    }
+  }
+}
+```
+
+## index.js
+
+```js
+/* @flow */
+
+import { CompositeDisposable } from 'atom'
+
+const myLinterPackage = {
```

Comment:
That's because according to ES2015 module's implementation by babel, it converts 

```js
export default {
  some: 'thing',
}
```
to
```
Object.defineProperty(exports, '_esModule', { value: true, enumerable: false })
Object.defineProperty(exports, '_default', { value: {
  some: 'thing',
}}, { enumerable: false })
```

The only reason this is working in Atom modules is because Atom is using babel 5.x and would break as soon as Atom upgrades to Atom 6.x or we decide to manually run babel like we did in linter-ui-default or linter-eslint